index.md

```diff
@@ -399,6 +400,24 @@ def third_function(
     ...
 ```
 
+#### 3.4 Tests
+
+- tests that assert query counts should include a comment that explains what is causing the queries.
+The comment should be able to explain, for each query, which table and for what purpose it was made.
+- Tip 1: Grab the raw SQL and understand/explain each SELECT
+- Tip 2: If updating a num queries test, get the SQL for what it used to be and for what you changed
+it to and diff the two to get to the bottom of exactly which SELECTs your change added/removed and
+update the comment.
+
+```python
+# 1 - template
+# 1 - conversation
+# 2 - employee, manager questions
+# 2 - employee, manager
```

Comment:
shouldnt this be like "get employee, manager"? feels a little vague as just "employee, manager" etc